session.go

```diff
@@ -52,7 +52,7 @@ func (c *Session) login(username, password string) error {
 
 	// login to API
 	params := map[string]string{
-		"user":     username,
+		"username":     username,
```

Comment:
This is not backward compatible for those using a pre-5.4 Zabbix. See https://github.com/zabbix-tools/go-zabbix/issues/54#issuecomment-1515727585